services/motion/builtin/builtin_test.go

```diff
@@ -698,811 +700,252 @@ func TestMoveOnMapTimeout(t *testing.T) {
-
+	gpsPoint := geo.NewPoint(0, 0)
 	dst := geo.NewPoint(gpsPoint.Lat(), gpsPoint.Lng()+1e-5)
-	expectedDst := r3.Vector{X: 380, Y: 0, Z: 0} // Relative pose to the starting point of the base; facing north, Y = forwards
 	epsilonMM := 15.
+	motionCfg := &motion.MotionConfiguration{PositionPollingFreqHz: 100, ObstaclePollingFreqHz: 1, PlanDeviationMM: epsilonMM}
 
-	t.Run("returns error when called with an unknown component", func(t *testing.T) {
-		t.Parallel()
-		injectedMovementSensor, _, _, ms := createMoveOnGlobeEnvironment(ctx, t, gpsPoint, nil, 5)
-		defer ms.Close(ctx)
-		success, err := ms.MoveOnGlobe(
-			ctx,
-			base.Named("non existent base"),
-			geo.NewPoint(0, 0),
-			0,
-			injectedMovementSensor.Name(),
-			nil,
-			nil,
-			nil,
-		)
-		test.That(t, err, test.ShouldBeError, errors.New("resource \"rdk:component:base/non existent base\" not found"))
-		test.That(t, success, test.ShouldBeFalse)
-	})
+	type testCase struct {
+		name            string
+		noise           r3.Vector
+		expectedSuccess bool
+		expectedErr     string
+		extra           map[string]interface{}
+	}
 
-	t.Run("returns error when called with an unknown movement sensor", func(t *testing.T) {
-		t.Parallel()
-		_, _, fakeBase, ms := createMoveOnGlobeEnvironment(ctx, t, gpsPoint, nil, 5)
-		defer ms.Close(ctx)
-		success, err := ms.MoveOnGlobe(
-			ctx,
-			fakeBase.Name(),
-			geo.NewPoint(0, 0),
-			0,
-			movementsensor.Named("non existent movement sensor"),
-			nil,
-			nil,
-			nil,
-		)
-		e := "Resource missing from dependencies. Resource: rdk:component:movement_sensor/non existent movement sensor"
-		test.That(t, err, test.ShouldBeError, errors.New(e))
-		test.That(t, success, test.ShouldBeFalse)
-	})
+	testCases := []testCase{
+		{
+			name:            "check we dont replan with a good sensor",
+			noise:           r3.Vector{Y: epsilonMM - 0.1},
+			expectedSuccess: true,
+			extra:           map[string]interface{}{"smooth_iter": 5},
+		},
+		// TODO(RSDK-5634): this should be uncommented when this bug is fixed
+		// {
+		// 	// This also checks that `replan` is called under default conditions when "max_replans" is not set
+		// 	name:            "check we fail to replan with a low cost factor",
+		// 	noise:           r3.Vector{Y: epsilonMM + 0.1},
+		// 	expectedErr:     "unable to create a new plan within replanCostFactor from the original",
+		// 	expectedSuccess: false,
+		// 	extra:           map[string]interface{}{"replan_cost_factor": 0.01, "smooth_iter": 5},
+		// },
+		{
+			name:            "check we replan with a noisy sensor",
+			noise:           r3.Vector{Y: epsilonMM + 0.1},
+			expectedErr:     fmt.Sprintf("exceeded maximum number of replans: %d", 4),
+			expectedSuccess: false,
+			extra:           map[string]interface{}{"replan_cost_factor": 10.0, "max_replans": 4, "smooth_iter": 5},
+		},
+	}
 
-	t.Run("returns error when request would require moving more than 5 km", func(t *testing.T) {
```

Comment:
Is this just getting deleted?  I dont see it being moved elsewhere

Reply:
Its here: https://github.com/viamrobotics/rdk/pull/3303/files/a530f9f1892ac34547952c8ed7f20e0fe6c4a104#diff-d9c3cfa3ab55320d976620b0fc7572b952eea9659a7c9433bb0243b8275d91feR1416

Reply:
Yeah I saw this when I got to that point in the file but forgot to delete my comment